docker-compose.yml

```diff
@@ -3,24 +3,26 @@ version: "3.7"
 services:
   mysql:
     image: mysql/mysql-server:5.7.31
-    restart: "no"
-    expose:
-      - "3306"
+    ports:
+      - "3306:3306"
```

Comment:
You dont want to use ports, then it's exposed on the host machine.

Reply:
Added requested changes